packages/babel-traverse/src/visitors.js

```diff
@@ -221,6 +221,8 @@ function wrapWithStateOrWrapper(oldVisitor, state, wrapper: ?Function) {
         newFn = function(path) {
           return fn.call(state, path, state);
         };
+        // Override toString in case this function is printed, we want to print the wrapped function, same as we do in `wrapCheck`
+        newFn.toString = () => fn.toString();
       }
 
       if (wrapper) {
```

Comment:
Can you override `toString` immediately before returning a `newFn`? Otherwise it will be overridden if `wrapper` is true.

Note that we have the same `toString` hack in `wrapCheck` so I consider it a bug fix.
https://github.com/babel/babel/blob/26eb8918702552c98bbe2dd9f41dd1d6aaeb7a1f/packages/babel-traverse/src/visitors.js#L261

Reply:
Yes, committed.

I did notice the hack, as mentioned in the comment I added.

Will change issue description to bug fix.